### DISCUSSION TITLE
Lesson 4 concept 5 subtitles

### DISCUSSION THREAD

**Discussion by bcebere** [OpenMined/courses#170] [Feedback] [answered]
Hello.

First of all, amazing work!

Just a small bug: the video for Lesson 4 concept 5 has no button for subtitles(sorry if it is already reported).

Keep up the good work!

**Comment by em-blue** [comment, marked as answer]
That's because we re-uploaded last night :-) youtube takes a little while to put the subtitles on - they should probably be up today or tomorrow. Great work getting to Lesson 5 already, I'm impressed!
